src/us/kbase/workspace/test/workspace/WorkspaceTest.java

```diff
@@ -8699,10 +8699,10 @@ public void maxReturnedObjectSize() throws Exception {
 			List<ObjectIdentifier> all = new LinkedList<ObjectIdentifier>();
 			all.addAll(ois1l2);
 			all.addAll(bothoi);
-			ws.setResourceConfig(build.withMaxReturnedDataSize(60).build());
+			ws.setResourceConfig(build.withMaxReturnedDataSize(80).build());
 			destroyGetObjectsResources(ws.getObjects(user, all));
-			ws.setResourceConfig(build.withMaxReturnedDataSize(59).build());
-			err = new IllegalArgumentException(String.format(errstr, 60, 59));
+			ws.setResourceConfig(build.withMaxReturnedDataSize(79).build());
+			err = new IllegalArgumentException(String.format(errstr, 80, 79));
```

Comment:
This change actually fixed a bug and so this test had to be adjusted. Previously, if a user requested data from the same object twice (which is possible - for example `myws/myobj` and `1/1/1` could point to the same object) it wasn't counted against the amount of data returned because the data was only pulled from the `BlobStore` once. Of course, the data is duplicated in the return and so the actual data package could've been larger than the 1GB limit. Now the data size is calculated based on the data returned to the user.